spec/motion-support/core_ext/time/conversion_spec.rb

```diff
@@ -1,50 +1,49 @@
-describe "Time" do
```

Comment:
OH I thought this was a comment on stabby procs!  Confused!

I agree; there's little advantage to having all this green and red, just for the sake of quotation preferences.